src/UXCapture.js

```diff
@@ -8,35 +9,49 @@ let _onMeasure;
 let _view;
 
 const UXCapture = {
+	_clearMarksAndMeasures: () => {
+		window.performance.clearMarks();
+		window.performance.clearMeasures();
+	},
+
 	/**
 	 * Sets `onMark` and `onMeasure` callbacks on UXCapture singleton
+	 * Also resets the view (supposed to be called once per page anyway)
 	 *
 	 * @param {object} config
 	 */
-	create: (config) => {
+	create: config => {
 		_onMark = config.onMark || NOOP;
 		_onMeasure = config.onMeasure || NOOP;
+		_view = null;
```

Comment:
What is the purpose of setting `_view=null` when view will already be `undefined`?

Reply:
Good question - it resets previously initialized instance of UXCapture returning it into page view mode, e.g. in tests where we run `UXCapture.create()` multiple times and as a singleton, it preserves the previous state (e.g. old view).

Do you feel that we should spell it out and provide alternative method of resetting the library?

Reply:
In the API implementation, you're only supposed to call `UXCapture.create()` once, so I don't think we should be adding additional logic to the implementation for the purpose of the writing tests. I believe if you just call `UXCapture.create()` in the `beforeAll()` implementation of the unit tests, it would work.